### PR TITLE
Automate OCP-37568/OCP-37569/OCP-27727 and some fix

### PR DESCRIPTION
**Pull request by duanwei33**
Automate snapshot with block volume and add cinder cases: 
[pass log](https://mastern-jenkins-csb-openshift-qe.apps.ocp4.prod.psi.redhat.com/job/Runner-v3-smoke/3274/console) with cinder
[pass log](https://mastern-jenkins-csb-openshift-qe.apps.ocp4.prod.psi.redhat.com/job/Runner-v3-smoke/3273/console) with aws

### CODE REVIEW

**Comment by duanwei33** [comment]
@chao007 @jhou1 Could you help take a look? Thanks.

File: features/storage/csi_snapshot.feature [hunk 1]
```diff
@@ -53,5 +53,71 @@ Feature: Volume snapshot test
...
+    Then the step should succeed
+    Given I obtain test data file "storage/misc/pod-with-block-volume.yaml"
+    When I run oc create over "pod-with-block-volume.yaml" replacing paths:
+      | ["metadata"]["name"]                                         | mypod-ori   |
+      | ["spec"]["volumes"][0]["persistentVolumeClaim"]["claimName"] | mypvc-ori   |
+      | ["spec"]["containers"][0]["volumeDevices"][0]["devicePath"]  | /dev/dblock |
+    Then the step should succeed
+    And the pod named "mypod-ori" becomes ready
+    When I execute on the pod:
+      | /bin/dd | if=/dev/zero | of=/dev/dblock | bs=1M | count=1 |
```
Comment (jhou1) [inline features/storage/csi_snapshot.feature:79]: Need a bit of explanation about what this step do.

Comment (duanwei33) [reply features/storage/csi_snapshot.feature:81]: Added comments to explain the steps.

**Pull request merged**